python/tests/integration/connectors/memory/test_weaviate_memory_store.py

```diff
@@ -67,10 +69,25 @@ def documents():
 
 @pytest.fixture
 def memory_store():
+    max_attempts = 5  # the number of retry attempts
+    delay = 5 * 60  # delay in seconds between each attempt
```

Comment:
Is the 60s number arbitrary? I'm a little concerned that this could result in a wait of 25 minutes before ultimately discovering a failing test. Could 30 seconds adequate?

Reply:
30s worked on my machine. Let's see if is adequate for the CI too.

fyi, the reason for the flakiness is weaviate/weaviate-python-client#349